content/en/blog/2025/otel-rocks.md

```diff
@@ -0,0 +1,49 @@
+
+## Why OTel Rocks
+
+Despite the challenges, our interviewees also acknowledged OTel's significant benefits.
+
+For instance, Alexandre from Pismo emphasized the importance of vendor neutrality. OTel allows them to consolidate all their data and switch to any vendor they want, without being locked into a specific ecosystem.
+
+James from Atlassian praised OTel's modular design, which allows users to replace small modular bits if something isn't working. This flexibility is especially crucial for power users who need to customize their setups.
+
+Elena from Delivery Hero shared how transformative it was to move from logs to distributed tracing with OTel. She described the experience of going back to logs as "inefficient." James also highlighted how the Java agent provides an "incredibly never-seen-before picture" of monolithic applications.
+
+Adriel, who is deeply involved in the OTel community, emphasized the value of open source and the ability to contribute. He noted that "once you push people past the point of it being hard and getting started with it the first time, it's like it's a breeze after that and it's super powerful."
+
+Alexandre from Pismo shared a compelling story of how implementing tail sampling with OTel drastically reduced their observability costs. They were able to optimize resource allocation and avoid over-scaling their collectors.
+
+Finally, Alexandre highlighted the role of OTel in bringing standardization to their observability practices. Before OTel, they had a mix of vendor-specific libraries and internal collectors, leading to inconsistencies and broken traces. OTel provided a "single library, a single standard" that significantly improved data quality.
+
+### It’s about the Community!
+
+The OpenTelemetry community is highly regarded by its members, with Adriel Perkins describing it as his first experience of being actively involved in open source, largely due to the enjoyable interactions with the people. He highlights the presence of many bright, kind, and helpful individuals who are always willing to offer guidance. The community members are also noted for their experience, which Adriel believes has personally helped him develop as an engineer. Furthermore, the community is characterized by being super helpful, passionate, and excited about the future of the project. While members may be direct at times, they are generally cordial and kind. James Moessis also points out that contributors can expect their code to be reviewed by dedicated individuals within the community, and that the developers are receptive to contributions. The existence of a contributor experience survey indicates the community's commitment to understanding and improving the contributor experience. Overall, the community fosters a supportive and collaborative environment where members can learn and grow.
```

Comment:
This is a rather chunky paragraph. Could we also split it by person, as in the previous sections?